src/auth.ts

```diff
@@ -123,6 +126,7 @@ const plugin = async (
             req.premium = payload.premium;
             req.roles = payload.roles;
             req.isTeamMember = payload.isTeamMember;
+            req.isPlus = payload.isPlus;
```

Comment:
```suggestion
            req.isPlus = !!payload.isPlus;
```

and then we can make it non optional in `AuthPayload`?

Reply:
Made this non-optional